context.go

```diff
@@ -158,7 +161,7 @@ func (cCtx *Context) lookupFlagSet(name string) *flag.FlagSet {
 			return c.flagSet
 		}
 	}
-
+	cCtx.onInvalidFlag(name)
```

Comment:
I would prefer if you move this call into Context.Value() . Thats the direct calling place from user so its better to be closer to there. lookupFlagSet is an internal function and so we shouldnt expect the user to get notified is we lookup something internally

Reply:
That won't work because the other getter functions don't call `Context.Value` internally.  https://github.com/urfave/cli/blob/v2.11.2/flag_uint.go#L75

Reply:
@dearchap any ideas on a better place to do the check?

Reply:
let me look at it.